web/src/app/pages/home/user/user.html

```diff
@@ -1,66 +1,61 @@
- <div class="row">
+<div class="row">
     <div class="col-md-12">
         <div ba-panel ba-panel-title=" " ba-panel-class="with-scroll">
-          <div class="side-message-navigation">
+            <div class="side-message-navigation">
 
 
- <div class="person-info row">
-          <div class="col-lg-4 col-md-12  no-padding">
-            <img ng-src="https://avatars1.githubusercontent.com/u/12439172?v=3&s=460" class="human-picture">
+                <div ng-init="getUser(234)" class="person-info row">
+                    <div class="col-lg-4 col-md-12  no-padding">
+                        <img ng-src="{{data.profile}}" class="human-picture">
 
-            <div class="name">
-              <h2 class="name-h">Niroshan</h2>
+                        <div class="name">
+                            <h2 class="name-h">{{data.firstname}}</h2>
 
-              <h2 class="name-h second-name">Ranapthi</h2>
+                            <h2 class="name-h second-name">{{data.lastname}}</h2>
```

Comment:
Use lower camel case for JavaScript variable declarations (https://en.wikipedia.org/wiki/CamelCase)
e.g.firstName lastName